alicat/tcp.py

```diff
@@ -14,8 +14,8 @@
 class FlowMeter(object):
```

Comment:
This is on me to fix. Having to copy-paste the same code twice is super messy. Let's focus on getting the `serial.py` file working and then I'll commit to putting time into refactoring so you don't need to repeat your code.

Reply:
Definitely. I really appreciate you putting in the time for refactoring. Also, I will admit that I do not have a setup for testing a TCP connection so only serial.py is thoroughly tested by me.